### PR TITLE
Change default theme from zoologist to twentytwentythree

### PR DESCRIPTION
**Pull request by autumnfjeld**
#### Proposed Changes

* Change the default theme for the build flow from Zoologist (fbhepr%2Skers%2Sjcpbz%2Qcho%2Qgurzrf%2Smbbybtvfg%2S-og) to Twentytwentythree (fbhepr%2Skers%2Sjcpbz%2Qcho%2Qgurzrf%2Sgjraglgjraglguerr%2S-og)

#### Testing Instructions
This is for the Build flow
* Using the Calypso Live link below
* Go to /start
* After you created a new site, you will land on the onboarding flow
* Continue until you land on the Design Picker
* On the Design Picker screen, don't pick a design. Instead, click "Skip for now" in the top right corner
* Check that the site theme is TT3 and that you have the correct starter content
* Check that the site's reading settings are correct: to "show posts"

#### Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #70604

### CODE REVIEW

**Comment by autumnfjeld** [comment]
This work is part of @autumnfjeld initiative to "code more". This will be slow going as lead duties take priority and I am having to re-learn a lot more than I anticipated. :(

**Comment by arthur791004** [comment]
@autumnfjeld I created a new site and it looks good! I assume we can simply merge this PR to change the default theme to TT3 🎉

Here is the screenshot of my new site

| Home | Pages | Posts |
| - | - | - |
| image | image | image |

There are some entry points to use Zoologist as the default theme, do we need to update them as well? See https://github.com/Automattic/wp-calypso/commit/2ac6eae1211dbc8e0ff4251ad1586e5918d7ecf0

**Review by arthur791004 — APPROVED** [review]

**Comment by autumnfjeld** [comment]
@arthur791004  I'll give this another test tomorrow, and I anticipate we'll want to make TT3 the default theme for write flow too.  Let's see what Ian says. p1681891512073159/1681796885.936569-slack-C048CUFRGFQ

**Comment by arthur791004** [comment]
Refer to pNEWy-gCz-p2#comment-56052, I plan to merge this PR to change the default theme of the newly created site.

File: client/lib/signup/site-type.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { find, get } from 'lodash';
 const getSiteTypePropertyDefaults = ( propertyKey ) =>
 	get(
 		{
-			theme: 'pub/zoologist',
+			theme: 'pub/twentytwentythree',
```
Comment (arthur791004) [inline client/lib/signup/site-type.js]: I guess we can remove it from client-side totally. I'll do it tomorrow and test it again!

Comment (arthur791004) [reply client/lib/signup/site-type.js]: Tested and it works well with D110177-code

**Pull request merged**